substrate/frame/revive/Cargo.toml

```diff
@@ -31,6 +31,7 @@ log = { workspace = true }
 serde = { optional = true, features = ["derive"], workspace = true, default-features = true }
 impl-trait-for-tuples = { workspace = true }
 rlp = { workspace = true }
+hex-literal = { workspace = true }
```

Comment:
No need for an additional dependency. Use `sp_core::hex2array` which does exactly the same thing.

Reply:
hex-literal macro is a better choice in this case since it will expand into a const rather than computing each time this is called

Reply:
Based on what the docs say about `sp_core::hex2array`, it is exactly the same as athei pointed out?

Reply:
Oh my bad. I was looking at `array_bytes::hex2array` since I found it on the pallet 😅